docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -157,6 +163,7 @@ enterpriseGateway:
   enabled: true
 gateway:
   enabled: false
+# MinIO storage configuration
 minio:
   enabled: true
```

Comment:
@zalegrala This is the GET MinIO values.

Reply:
@zalegrala MinIO section updated with the values for the user and password, as well as buckets.

Reply:
This looks good and matches below for the `storage.admin` section.